main.go

```diff
@@ -2,8 +2,124 @@ package main
 
 import (
 	"fmt"
+	"os"
+
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/microkit/command"
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/microkit/transaction"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+
+	"github.com/giantswarm/prometheus-config-controller/flag"
+	"github.com/giantswarm/prometheus-config-controller/server"
+	"github.com/giantswarm/prometheus-config-controller/service"
+)
+
+var (
+	f *flag.Flag = flag.New()
+
+	description string = "The prometheus-config-controller provides Prometheus service discovery for Kubernetes clusters on Kubernetes."
+	gitCommit   string = "n/a"
+	name        string = "prometheus-config-controller"
+	source      string = "https://github.com/giantswarm/prometheus-config-controller"
 )
 
+func panicOnErr(err error) {
+	if err != nil {
+		panic(fmt.Sprintf("%#v", err))
+	}
+}
```

Comment:
Lets please do it with proper error handling. See https://github.com/giantswarm/draughtsman-operator/blob/5319e685b8400886056b8b4ba188c5e8c3c4a9bb/main.go#L35-L40.

Reply:
Done